setup.js

```diff
@@ -143,6 +143,11 @@ const beginSetup = async (preferences) => {
     }
   });
 
+  bundler.stderr.on('data', (data) => {
+    log(chalk.red('Error:'));
+    log(String(data));
+  });
+            
   bundler.on('exit', function (code, signal) {
```

Comment:
Thank you for your MR @paramaggarwal!

There's this code already to print out errors if the server process exits with an error code. Is there a specific error you faced that this missed out?

Reply:
If `jekyll start` throws an error, then it does not show it on console. Just says that error occurred.